src/containers/Article.js

```diff
@@ -85,6 +100,28 @@ class Article extends PureComponent {
           event: 'gtm.load',
         },
       })
+      this.startTimer()
+    }
+    if (this.state.isReachedTargetHeight && this.state.isReachedTargetTime) {
+      // TODO: waiting for api
+      // eslint-disable-next-line no-undef
+      alert(`你閱讀了 post_id: ${_.get(this.props.post, 'id')} 喔喔喔`)
```

Comment:
XD
alert `slug` may be easier to read & match (nice to have)

Reply:
fixed